components/Card/Card.stories.js

```diff
@@ -0,0 +1,20 @@
+import { storiesOf } from '@storybook/vue'
+import Card from './Card.vue'
+
+storiesOf('Card', module)
+  .add('default', () => ({
+    components: { Card },
+    template: `<Card>TEST</Card>`
+  }))
+  .add('without shadow ', () => ({
+    components: { Card },
+    template: `<Card noShadow >TEST</Card>`
```

Comment:
FYI: you can use `no-shadow` instead of `noShadow` which is nicer to read and still interpreted correctly by vue